code/modules/spells/spell_types/shapeshift/_shape_status.dm

```diff
@@ -190,6 +190,10 @@
 	if(!QDELETED(source_spell) && source_spell.owner == owner)
 		source_spell.Grant(caster_mob)
 
+	// Prevent round removal by consuming stuff when shapeshifted
+	for(var/atom/movable/thing as anything in owner.contents)
```

Comment:
```suggestion
	for(var/atom/movable/thing as anything in owner.get_equipped_item(true) + owner.held_items + implants)
```
I think this covers most stuff.

Reply:
The issue here is stuff like becoming a space dragon then eating someone.
![image](https://github.com/tgstation/tgstation/assets/49160555/fc02ad83-2211-4ea2-9ba2-a5b077241542)

And yes, these are directly in contents

Reply:
OH!

Reply:
well then, consider this resolved then.